hyppo/independence/hhg.py

```diff
@@ -224,3 +309,97 @@ def _pearson_stat(distx, disty):  # pragma: no cover
                     S[i, j] = ((n - 2) * (t12 * t21 - t11 * t22) ** 2) / denom
 
     return S
+
+
+def _point_distance(self, x, y, zx, zy, **kwargs):
```

Comment:
how is this any different than the `compute_distance` function in every other `hyppo` function

Reply:
I believe that I was using this method since before there were two types of center point and each type required its adjustment to use compute_distance properly, so I used cdist instead to unify the distance calc.

Since I'm just doing center of mass now, I'll make the necessary adjustment for compute_distance.

Reply:
Function removed and code adjusted for compute_distance.